### DISCUSSION TITLE
High Privilege on secrets-store container

### DISCUSSION THREAD

**Discussion by rasrivastava-als** [Azure/secrets-store-csi-driver-provider-azure#1197] [Q&A]
container "secrets-store" under csi-secret-store pod has prvilege as true? Is it possible to set this privilege to specific UID or set it as false?

**Comment by aramase** [comment]
There is a note in the docs on why `privileged: true` is required for the driver: https://secrets-store-csi-driver.sigs.k8s.io/concepts.html#security.
